frontend/src/components/user/MyProfile.js

```diff
@@ -135,52 +177,106 @@ const MyProfile = ({ user }) => {
       }
     }
   };
+
   return (
-    <div>
-      {alert.show && (
-        <Alert severity={alert.severity} onClose={() => setAlert({ ...alert, show: false })}>
-          {alert.message}
-        </Alert>
-      )}
-      <Grid item xs={6}>
-        <form className={classes.form} noValidate>
-          <InputField values={username} label={userData.username} />
-          <InputField values={firstName} label={userData.firstname} />
-          <InputField values={lastName} label={userData.lastname} />
-          <InputField values={email} label={userData.email} />
-          <InputField
-            values={oldPassword}
-            label="current password"
-            autocomplete="old-pwd"
-            required={true}
-          />
-          <InputField values={password} label="password" autocomplete="new-pwd" />
-          <InputField values={confirmPassword} label="confirm password" autocomplete="new-pwd" />
-          <Autocomplete
-            id="language"
-            value={lang}
-            options={langOptions}
-            getOptionLabel={(option) => option.label}
-            getOptionSelected={(option, value) => option.value === value.value}
-            className={classes.select}
-            onChange={(event, value) => {
-              if (value !== null) setLang(value);
-            }}
-            renderInput={(params) => (
-              <TextField required {...params} label="Select language" variant="outlined" />
-            )}
-          />
-        </form>
-      </Grid>
-      <Button variant="outlined" color="secondary" onClick={handleClick}>
-        Save
-      </Button>
-    </div>
+    userData && (
+      <Container component="main" className={classes.paper} maxWidth="sm">
+        <CssBaseline />
+        <Typography component="h2" variant="h4" gutterBottom>
+          {t('myProfile.title')}
+        </Typography>
+
+        <Grid container spacing={6}>
+          <Grid item xs={12} sm={5} className={classes.column}>
+            <UpdatePhoto avatar={avatar} setAvatar={setAvatar} />
+
+            <Button
+              type="submit"
+              variant="outlined"
+              color="primary"
+              onClick={userProfileModal.handleClickOpen}
+            >
+              User Profile
+            </Button>
+            <p>(this button will be removed)</p>
+          </Grid>
+          <Grid item xs={12} sm={7} className={classes.column}>
+            <form className={classes.form} noValidate>
+              <InputField
+                values={username}
+                label={`${t('form.username')}: ${userData.username}`}
+              />
```

Comment:
I really liked how you changed the labels for input forms, they became more clean and readable